cloud/ec2_test.go

```diff
@@ -685,7 +685,19 @@ func (s *EC2Suite) TestStopInstance() {
 
 	s.Error(s.onDemandManager.StopInstance(ctx, hosts[0], evergreen.User))
 	s.Error(s.onDemandManager.StopInstance(ctx, hosts[1], evergreen.User))
+
+	manager, ok := s.onDemandManager.(*ec2Manager)
+	s.True(ok)
+	mock, ok := manager.client.(*awsClientMock)
+	s.True(ok)
+	mock.Instance = &ec2.Instance{
+		InstanceId: aws.String("host-running"),
+		State: &ec2.InstanceState{
+			Name: aws.String("stopped"),
+		},
+	}
```

Comment:
This seems like a strange way to test the mock. I didn't trace it back but shouldn't the mock function itself be setting the state to stopped?

Reply:
You're right, fixed!

Reply:
Should StartInstances be something similar?

Reply:
I think so, originally didn't have to do anything because the mock GetInstanceInfo always returns an instance with the state "running" but definitely a good idea to fix StartInstance in a similar way!

Reply:
LGTM